scripts/fix_db/current.py

```diff
@@ -1,8 +1,3 @@
-"""
-Loop over jobs.
-If the corresponding project still exists, set the project_dump.
-"""
-import json
 import logging
```

Comment:
General comment on this file: there a bunch of commented lines. Are they needed? If not, let's remove them.

Reply:
Commented lines are intended to temporarly skip this two validations
```python
WorkflowRead(**workflow.model_dump())

DatasetRead(**dataset.model_dump())
```
By introducing them, we get this error in `migrations.yml`
```shell
Traceback (most recent call last):
  File "/home/runner/work/fractal-server/fractal-server/scripts/fix_db/current.py", line 55, in <module>
    WorkflowRead(**workflow.model_dump())
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 2 validation errors for WorkflowRead
task_list
  field required (type=value_error.missing)
project
  field required (type=value_error.missing)
Error: Process completed with exit code 1.
```
it seems like we are not loading the relationships along with the main objects.

Reply:
This is a known issue: https://github.com/fractal-analytics-platform/fractal-server/issues/489. It's likely that there are more elegant solutions (and they will possibly change in Pydantic v2), but I think the following logic may already work (to check).

```python
from sqlalchemy import select
from fractal_server.app.db import get_sync_db
from fractal_server.app.models import Workflow
from fractal_server.app.schemas import WorkflowRead


with next(get_sync_db()) as db:
    stm = select(Workflow)
    workflows = db.execute(stm).scalars().all()
    for workflow in workflows:
        assert workflow.task_list is not None
        assert workflow.project is not None

        # Option 1 (fails)
        WorkflowRead(**workflow.model_dump())
        
        # Option 2 (goes through)
        project = workflow.project
        task_list = workflow.task_list
        WorkflowRead(**workflow.model_dump(exclude={"task_list", "project"}),  task_list=task_list, project=project)
```